### PR TITLE
Button click

### PR DESCRIPTION
**Pull request by zerox1990**
Apparently the reason the files said they were changed, not looking at the lines of code I added for the audio clip, was because the line endings were inconsistent or something and I said Windows when the prompt came up so I don't think it actually affected anything.

### CODE REVIEW

File: SGoopas/Assets/Scripts/Game/Triggers/ButtonInteractTrigger.cs [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
  * Will cause a button to scale when you press it.
- */
+ */
 public class ButtonInteractTrigger : InteractTrigger {
     private Vector3 originalScale;
-    private bool triggerOn = false;
-
+    private bool triggerOn = false;
+    public AudioClip audioClip;
+    public AudioSource audioSource;
```
Comment (dannyflax) [inline SGoopas/Assets/Scripts/Game/Triggers/ButtonInteractTrigger.cs]: Can you make it so you don’t have to set this up in the editor? Ideally the buttons will just work, you shouldn’t have to set the sound every time for each button.

**Comment by dannyflax** [comment]
The button click should be setup in MasterMonoBehaviour, that way you can play the sound from there in any scene _without_ having to set it up for every object individually.

**Comment by dannyflax** [comment]
also, we already have a click sound file. no need to import a new one.

**Comment by zerox1990** [comment]
So, set up variables in MasterMono?

**Comment by zerox1990** [comment]
You mean MenuPlayer.cs? Only MainMenu script I could find was MainMenuState.cs. This was on master so is it on another branch?

**Pull request closed without merging**